### DISCUSSION TITLE
Unable to configure a SLURM cluster on a shared vpc using the google cloud hpc toolkit

### DISCUSSION THREAD

**Discussion by irishwithag91** [GoogleCloudPlatform/cluster-toolkit#1279] [General]
Attempted to create a SLURM cluster on a shared VPC using the following template 
https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/community/examples/hpc-cluster-small-sharedvpc.yaml

Followed the pre requisites required for filestore creation 

The login and control nodes were created successfully 
After ssh into the login nodes , and running the sinfo command 
Getting the following error : 

[slurm-hpc-small-shared-vpc-login0 ~]$ sinfo
sinfo: error: resolve_ctls_from_dns_srv: res_nsearch error: Unknown host
sinfo: error: fetch_config: DNS SRV lookup failed
sinfo: error: _establish_config_source: failed to fetch config
sinfo: fatal: Could not establish a configuration source

Can someone please help on how to debug this further

**Comment by mr0re1** [comment]
We're looking at this issue. Could you please provide version of toolkit you were using (`./ghpc --version`)?

**Reply by irishwithag91** [reply]
Thanks @mr0re1  , please find the version number : 

ghpc version v1.16.0
Built from 'main' branch.
Commit info: v1.16.0-0-g8bac6eca-dirty

**Comment by mr0re1** [comment]
Hello @irishwithag91 , I wasn't able to reproduce this problem.
I've used same  `ghpc v.1.16.0` and `community/examples/hpc-cluster-small-sharedvpc.yaml` and it works as expected.

Did you set proper `host_project_id`, `network_name`, and `subnetwork_name` as advised in the example comments?
In my attempt to reproduce it I've used default network:
```
network_name: default
subnetwork_name: default
```
